src/CustomComponents/NavBar.css

```diff
@@ -0,0 +1,89 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Montserrat',sans-serif;
+}
+
+nav {
```

Comment:
Please be consistent with your {}. In some cases you have a space after the class while others you don't. Just stay consistent so its either nav{ or nav {    This applies to this entire file